docs/products/flink/howto/manage-flink-tables.rst

```diff
@@ -0,0 +1,74 @@
+Manage tables in Apache Flink® applications
+===========================================
+
+Aiven for Apache Flink® allows you to map source and target data structures as `Flink tables <https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/sql/create/#create-table>`_ and use transformation statements to transform data. Some of the operations you can perform on a table include:
+
+* Import existing tables
+* Add new tables
+* Clone tables
+* Edit tables
+* Delete tables
+
+Before performing any operation on a table in a Flink application, you must stop the application. To stop an application, navigate to the **Applications** tab on your Aiven for Apache Flink® service, select the desired application from the list, and click **Stop Deployment**.
```

Comment:
is this true @HelenMel ? Can't you edit while an app is running?

Reply:
> is this true @HelenMel ? Can't you edit while an app is running?

At least when I tried on an application that is running, you can only view the table. You cannot edit. To edit or make changes, I had to stop the application and create a new version (I need to mention this in the doc.)
But @HelenMel, it would be great if you could confirm this. Thanks.